app/views/documents/edit.html.erb

```diff
@@ -17,27 +17,28 @@
       } %>
 
       <% url_preview = capture do %>
-        <p class="govuk-heading-s">The web address for this content will look like:</p>
+        <p class="govuk-heading-s"><%= t("pages.documents.edit.url_preview.available") %></p>
 
-        <span id="no-title-id" class="govuk-body <%= 'app-hidden' unless @document.base_path.nil? %>">You haven’t entered a title yet.</span>
+        <span id="no-title-id" class="govuk-body <%= 'app-hidden' unless @document.base_path.nil? %>"><%= t("pages.documents.edit.url_preview.no_title") %></span>
 
-        <div id="url-preview-id" <%= 'class="app-hidden"' if @document.base_path.nil? %>>
-            <span class="govuk-body">www.gov.uk</span><span id="base-path-id" class="govuk-body" ><%= "#{@document.base_path}" %></span>
+        <div id="url-preview-id" <%= 'class="app-hidden"' if @document.base_path.nil? %>
+          <span class="govuk-body"><%= Plek.new.external_url_for("www") %></span><span id="base-path-id" class="govuk-body" ><%= "#{@document.base_path}" %></span>
         </div>
 
-        <span id="error-generating-path-id" class="govuk-body app-hidden">Unable to preview address, please edit title and try again.</span>
+        <span id="error-generating-path-id" class="govuk-body app-hidden"><%= t("pages.documents.edit.url_preview.error") %></span>
       <% end %>
 
       <%= render "govuk_publishing_components/components/inset_text", {
         text: url_preview.html_safe
       } %>
     </div>
+
     <div class="govuk-grid-column-one-third">
       <%= render "components/contextual-guidance", {
         id: "document-title-guidance",
-        title: "Writing a news title"
+        title: t("formats.document_types.#{@document.document_type}.guidance.title.title")
       } do %>
-        <p class="govuk-body">The title must make clear what the content offers users. Use the words your users do to help them find this. Avoid wordplay or teases.</p>
+        <p class="govuk-body"><%= t("formats.document_types.#{@document.document_type}.guidance.title.body") %></p>
```

Comment:
I understand guidance on a field could vary on document_type but I don't believe we have this need right now? maybe we can place the guidance further up the hierarchy, that could go some way to avoiding the need to define translations as part of the factory.

Reply:
I understand it's going to vary on most things. Hopefully this is now less of a concern now the format-specific guidance is in document_types.yml